### PR TITLE
docs(color_thresholds): explain resulting colors

### PR DESCRIPTION
**Pull request by ildar170975**
https://github.com/kalkih/mini-graph-card/issues/894

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,10 @@ color_thresholds:
   - value: 4
     color: "#0000ff"
 ```
+i.e. if value is between "0" (including this value) & "1.33333" - color is "#ff0000",
```
Comment (akloeckner) [inline README.md:205]: This looks quite "like a block", when rendered. Could you maybe make it a list?  I think, that would be more readable.

**Review by akloeckner — COMMENTED** [review]
Thanks! Could you base this PR towards the `dev` branch, please? (Maybe it'll be time to merge `dev` into `master`, afterwards.)

I also added a comment to the new text.

File: README.md (hunk 1 above)
Comment (ildar170975) [reply README.md:205]: Could you give an example?

File: README.md [hunk 2]
```diff
@@ -202,6 +202,10 @@ color_thresholds:
...
+between "1.33333" (including this value) & "2.666667" - color is "#ffff00",
+between "2.666667" (including this value) & "4" - color is "#00ff00",
+equal to or more than "4" - color is "#0000ff".
```
Comment (akloeckner) [inline README.md:208]: I think, this change should make the resulting colors more readable:

```suggestion
The example above will result in the following colors of the graph: if value is 
* between `0` (including this value) and  `1.33333`, the color is `#ff0000`,
* between `1.33333` (including this value) and `2.666667`, the color is `#ffff00`,
* between `2.666667` (including this value) and `4`, the color is `#00ff00`,
* equal to or more than `4`, the color is `#0000ff`.
```

What do you think? (If I got it right, you can just "apply" or "accept" the suggestion, if you agree.)

**Review by akloeckner — COMMENTED** [review]
I used the suggestion-feature to give an example for the list, I was refering to. I couldn't see this as rendered markdown in the comment, but I think, it should be the right syntax.

We  should also fix the PR title to something semantic, e.g. `docs(color_thresholds): explain resulting colors`. That way, we will have a semantic commit title, when we "squash and merge". And that will result in the release being bumped properly.

File: README.md (hunk 2 above)
Comment (ildar170975) [reply README.md]: Good, I think

Comment (ildar170975) [reply README.md]: > fix the PR title 

Done!

**Comment by akloeckner** [comment]
Looks great! I'll merge it. Thanks!

**Pull request merged**